app/components/MeetupButton.tsx

```diff
@@ -0,0 +1,21 @@
+import MeetupIcon from "~/components/icons/MeetupIcon";
+import type { PropsWithChildren } from "react";
+
+export default function MeetupButton({
+  link,
+  children,
+}: PropsWithChildren<{ link: string }>) {
```

Comment:
Nit: I prefer simply:

```
{
  link: string;
  children: React.ReactNode
}
```

Mostly since the child is actually required.

Reply:
interesting. i don't think i ever realized that was optional. i'd argue `PropsWithChildren` is a misleading name then, but that's neither here nor there. 😉 

fixed via:
```typescript
type PropsWithRequiredChildren<P = unknown> = P & {children: ReactNode};
```

Reply:
Haha I would agree with that argument. I really don't like the default `children` as optional thing, since most custom components that take in children really want them to be required.

Thanks Colby!